app/controllers/api/v3/sync_controller.rb

```diff
@@ -14,7 +14,6 @@ def __sync_to_user__(response_key)
     records = records_to_sync
 
     AuditLog.create_logs_async(current_user, records, "fetch", Time.current) unless disable_audit_logs?
-    log_block_level_sync_metrics(response_key)
```

Comment:
Is some form of this logging still valuable? If our regular sync logging has enough detail (sync region ID for every sync request), then we should be good.

Reply:
We get sufficient information through the headers. This was mainly feeds the [datadog dashboard](https://app.datadoghq.com/dashboard/5gh-pzf-66k/block-level-sync-india-production), which I think we can remove now (haven't looked at it in a _long time_).